server/lib/tuist/qa/agent.ex

```diff
@@ -3,33 +3,133 @@ defmodule Tuist.QA.Agent do
   Tuist QA agent module.
   """
 
+  alias LangChain.Chains.LLMChain
```

Comment:
I remember asking this somewhere, but I can't find it anymore. This will eventually be part of the standalone executable that we'll run in Namespace, no?